src/integrationTest/resources/application-integration.yaml

```diff
@@ -0,0 +1,2 @@
+hmc:
+  url: http://localhost:10010
```

Comment:
Is this being used anywhere ?

Reply:
Nope, just needed for FeignClientErrorDecoderIntegrationTest